src/java.base/unix/native/libnet/SdpSupport.c

```diff
@@ -95,19 +95,27 @@ Java_sun_net_sdp_SdpSupport_convert0(JNIEnv *env, jclass cls, int fd)
 
         /* copy socket options that are relevant to SDP */
         len = sizeof(arg);
-        if (getsockopt(fd, SOL_SOCKET, SO_REUSEADDR, (char*)&arg, &len) == 0)
-            setsockopt(s, SOL_SOCKET, SO_REUSEADDR, (char*)&arg, len);
+        if (getsockopt(fd, SOL_SOCKET, SO_REUSEADDR, (char*)&arg, &len) == 0) {
+            res = setsockopt(s, SOL_SOCKET, SO_REUSEADDR, (char*)&arg, len);
+            if (res < 0) JNU_ThrowIOExceptionWithLastError(env, "setsockopt SO_REUSEADDR");
```

Comment:
OFED deprecated SDP a long time and I think we should just remove this code from the JDK. The copying of the socket options was always a best effort basis. It requires Infiniband config to be able to test these changes so maybe it would be better to just drop it from this change as it can't be tested.

Reply:
> Hi Alan, do you mean "drop this chunk from the change" or "drop the coding that is trying to copy the socket options" ?

Yes, I think drop SdpSupport.c from the PR. We probably should have removed the SDP support a long time ago.